runtime/shiden/src/lib.rs

```diff
@@ -288,6 +290,8 @@ impl pallet_dapps_staking::Config for Runtime {
     type MinimumRemainingAmount = MinimumRemainingAmount;
     type HistoryDepth = HistoryDepth;
     type BonusEraDuration = BonusEraDuration;
+    type MaxUnlockingChunks = MaxUnlockingChunks;
+    type UnbondingPeriod = UnbondingPeriod;
```

Comment:
Agree

Reply:
This wasn't done because I believe we should first test it on Shiden - this is just the branching issue we've discussed.
Since pallet was changed and new configurable constants were added they need to be defined for all runtimes using that pallet. That includes Shiden. Otherwise the code won't compile.